cmdstanpy/utils/cmdstan.py

```diff
@@ -17,6 +18,44 @@
 EXTENSION = '.exe' if platform.system() == 'Windows' else ''
 
 
+def determine_linux_arch() -> str:
+    machine = platform.machine()
+    arch = ""
+    if machine == "aarch64":
+        arch = "arm64"
+    elif machine == "armv7l":
+        # Telling armel and armhf apart is nontrivial
+        # c.f. https://forums.raspberrypi.com/viewtopic.php?t=20873
+        if subprocess.run(
+            ["readelf -A /proc/self/exe | grep Tag_ABI_VFP_args"],
+            shell=True,
+            check=False,
```

Comment:
Does this command also fail sometimes?

Reply:
`grep` sets the return code to 1 when it doesn’t find anything (which is why we’re branching on it), so using check would be a problem 

I don’t believe it would fail, since readelf and grep are both standard Unix tools which should be available